Sergey_Shashin/src/components/MessageForm/MessageForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class MessageForm extends Component {
+  state = {
+    author: '',
+    text: ''
+  }
+
+  static propTypes = {
+    onSend: PropTypes.func,
+  }
+
+  handleMessageSend = () => {
+    const { onSend } = this.props;
+
+    if (typeof onSend === 'function') {
+      onSend(this.state);
+
+      this.setState({
+        text: ''
+      });
+    }
+
+
+  }
+
+  handleInputChange = (event) => {
+    const FieldName = event.target.name;

```

Comment:
с заглавной буквы обычно именуются классы
```suggestion
    const  fieldName = event.target.name;
```